### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by Adamws33**
installed, and configured eslint
ran fix command on all files 
updated each file (not currently in dev) and corrected any errors that were not fixable 
refactored smaller files while correcting lint errors
identified other refactor issues and added to the dst refactor ticket.

### CODE REVIEW

File: src/components/CropSelector/CropDetailsModal.js [hunk 1]
```diff
@@ -36,9 +38,9 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const CropDetailsModalComponent = (props) => {
+function CropDetailsModalComponent(props) {
   const { state } = useContext(Context);
-  let crop = props.crop;
+  const { crop, setModalOpen, modalOpen } = props;
```
Comment (mikahpinegar) [inline src/components/CropSelector/CropDetailsModal.js:43]: did the linter do this? can we make it put props in the arg?

File: src/components/CropSelector/CropSidebar/Components/PlantHardinessZone.js [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+  Chip, Collapse, Grid, List, ListItem, ListItemText, Typography,
+} from '@mui/material';
 import { ExpandLess, ExpandMore } from '@mui/icons-material';
 import React, { Fragment, useContext } from 'react';
 import { Context } from '../../../../store/Store';
 
-const PlantHardinessZone = (props) => {
-  let { handleToggle, dispatch, sfilters } = props;
+function PlantHardinessZone(props) {
+  const { handleToggle, dispatch, sfilters } = props;
```
Comment (mikahpinegar) [inline src/components/CropSelector/CropSidebar/Components/PlantHardinessZone.js:9]: another prop situation

File: src/components/CropSelector/CropSidebar/Components/PreviousCashCrop.js [hunk 3]
```diff
@@ -18,14 +18,14 @@ import { Context } from '../../../../store/Store';
 import DateRangeDialog from '../../DateRangeDialog';
 import { CustomStyles } from '../../../../shared/constants';
 
-const PreviousCashCrop = (props) => {
-  let { handleToggle, classes, setDateRange } = props;
+function PreviousCashCrop(props) {
+  const { handleToggle, classes, setDateRange } = props;
```
Comment (mikahpinegar) [inline src/components/CropSelector/CropSidebar/Components/PreviousCashCrop.js:22]: prop

File: src/components/CropSelector/CropSidebar/Components/SidebarFilter.js [hunk 4]
```diff
@@ -1,11 +1,13 @@
...
+} from '@mui/material';
 import { ExpandLess, ExpandMore } from '@mui/icons-material';
 import React, { Fragment, useContext } from 'react';
 import ForwardFilter from '../../Filters/ForwardFilter';
 import { Context } from '../../../../store/Store';
 
-const SidebarFilter = (props) => {
-  let {
+function SidebarFilter(props) {
+  const {
```
Comment (mikahpinegar) [inline src/components/CropSelector/CropSidebar/Components/SidebarFilter.js:10]: prop

File: src/components/InformationSheet/InformationSheetDictionary.js [hunk 5]
```diff
@@ -5,25 +5,27 @@
...
+} from 'react';
 import z4Dict from '../../shared/json/zone4/data-dictionary.json';
 import z5Dict from '../../shared/json/zone5/data-dictionary.json';
 import z6Dict from '../../shared/json/zone6/data-dictionary.json';
 import z7Dict from '../../shared/json/zone7/data-dictionary.json';
 import { Context } from '../../store/Store';
 import DictionaryContent from './DictionaryContent';
 
-const InformationSheetDictionary = (props) => {
+function InformationSheetDictionary(zone, from) {
```
Comment (mikahpinegar) [inline src/components/InformationSheet/InformationSheetDictionary.js:18]: destructure props?

File: src/components/MyCoverCropList/MyCoverCropComparisonComponent.js [hunk 6]
```diff
@@ -56,12 +56,12 @@ const lightBG = {
   minHeight: '36px',
 };
 
-const MyCoverCropComparisonComponent = (props) => {
+function MyCoverCropComparisonComponent(props) {
```
Comment (mikahpinegar) [inline src/components/MyCoverCropList/MyCoverCropComparisonComponent.js:59]: props

File: src/components/MyCoverCropList/MyCoverCropList.js [hunk 7]
```diff
@@ -8,15 +9,14 @@ import { Button, Typography } from '@mui/material';
...
 import React, { Fragment, useContext, useEffect } from 'react';
 import { useHistory } from 'react-router-dom';
+import ReactGA from 'react-ga';
 import { Context } from '../../store/Store';
 import MyCoverCropCardsComponent from './MyCoverCropCardsComponent';
 import MyCoverCropComparisonComponent from './MyCoverCropComparisonComponent';
-import ReactGA from 'react-ga';
 
-const MyCoverCropList = (props) => {
+function MyCoverCropList(comparisonView, from) {
```
Comment (mikahpinegar) [inline src/components/MyCoverCropList/MyCoverCropList.js:17]: destructure

**Review by mikahpinegar — CHANGES_REQUESTED** [review]

File: src/components/CropSelector/Filters/Filters.js [hunk 8]
```diff
@@ -130,16 +129,16 @@ const Tip = ({ filter, omitHeading }) => {
   );
 }; // Tip
 
-const Filters = forwardRef(({ props }, ref) => {
+const Filters = forwardRef(({ props }) => {
   const { state, dispatch } = useContext(Context);
 
-  let { filters, setSidebarFilterOptions, sidebarFilterOptions } = props;
+  const { filters, setSidebarFilterOptions, sidebarFilterOptions } = props;
```
Comment (mikahpinegar) [inline src/components/CropSelector/Filters/Filters.js:135]: props

File: src/components/InformationSheet/InformationSheetDictionary.js [hunk 9]
```diff
@@ -5,25 +5,27 @@
...
+} from 'react';
 import z4Dict from '../../shared/json/zone4/data-dictionary.json';
 import z5Dict from '../../shared/json/zone5/data-dictionary.json';
 import z6Dict from '../../shared/json/zone6/data-dictionary.json';
 import z7Dict from '../../shared/json/zone7/data-dictionary.json';
 import { Context } from '../../store/Store';
 import DictionaryContent from './DictionaryContent';
 
-const InformationSheetDictionary = (props) => {
+const InformationSheetDictionary = (zone, from) => {
```
Comment (mikahpinegar) [inline src/components/InformationSheet/InformationSheetDictionary.js:18]: destructure

**Review by mikahpinegar — CHANGES_REQUESTED** [review]

File: src/components/InformationSheet/InformationSheetDictionary.js (hunk 9 above)
Comment (Adamws33) [reply src/components/InformationSheet/InformationSheetDictionary.js:18]: done

File: src/components/CropSelector/Filters/Filters.js (hunk 8 above)
Comment (Adamws33) [reply src/components/CropSelector/Filters/Filters.js:135]: done

File: src/components/CropSelector/CropSidebar.js [hunk 10]
```diff
@@ -51,17 +55,16 @@ const useStyles = makeStyles((theme) => ({
...
-
+const CropSidebarComponent = ({
+  comparisonView,
+  isListView,
+  from,
+  setGrowthWindow,
+  toggleComparisonView,
+  toggleListView,
+  style,
+  props,
```
[the remark this answers is not in the document]
Comment (Adamws33) [reply src/components/CropSelector/CropSidebar.js:66]: they seem to be used

File: src/components/CropSelector/CropSelector.js [hunk 11]
```diff
@@ -49,14 +52,14 @@ const ScrollTop = (props) => {
 
 const CropSelector = (props) => {
```
[the remark this answers is not in the document]
Comment (Adamws33) [reply src/components/CropSelector/CropSelector.js:53]: done

File: src/components/CoverCropExplorer/ConsentModal.js [hunk 12]
```diff
@@ -4,8 +4,10 @@
...
 
-import { Modal, Box, Typography, Button, Grid } from '@mui/material';
-import React, { Fragment, useContext, useState } from 'react';
+import {
+  Modal, Box, Typography, Button, Grid,
+} from '@mui/material';
+import React, { useContext, useState } from 'react';
 import { Context } from '../../store/Store';
 
 const ExplorerCardView = (props) => {
```
[the remark this answers is not in the document]
Comment (Adamws33) [reply src/components/CoverCropExplorer/ConsentModal.js:13]: done

File: src/components/MyCoverCropList/MyCoverCropList.js (hunk 7 above)
Comment (Adamws33) [reply src/components/MyCoverCropList/MyCoverCropList.js]: done

File: src/components/MyCoverCropList/MyCoverCropComparisonComponent.js (hunk 6 above)
Comment (Adamws33) [reply src/components/MyCoverCropList/MyCoverCropComparisonComponent.js]: this is one of the not yet completed components yet to be worked.

File: src/components/InformationSheet/InformationSheetDictionary.js (hunk 5 above)
Comment (Adamws33) [reply src/components/InformationSheet/InformationSheetDictionary.js]: done

File: src/components/CropSelector/CropSidebar/Components/SidebarFilter.js (hunk 4 above)
Comment (Adamws33) [reply src/components/CropSelector/CropSidebar/Components/SidebarFilter.js:10]: doen

File: src/components/CropSelector/CropSidebar/Components/PreviousCashCrop.js (hunk 3 above)
Comment (Adamws33) [reply src/components/CropSelector/CropSidebar/Components/PreviousCashCrop.js:22]: done

File: src/components/CropSelector/CropSidebar/Components/PlantHardinessZone.js (hunk 2 above)
Comment (Adamws33) [reply src/components/CropSelector/CropSidebar/Components/PlantHardinessZone.js:9]: done

File: src/components/CropSelector/CropDetailsModal.js (hunk 1 above)
Comment (Adamws33) [reply src/components/CropSelector/CropDetailsModal.js:43]: nope it just didnt catch files that didnt have props. issues so some of these still lingered glad you caught them

**Review by mikahpinegar — APPROVED** [review]

**Pull request merged**